plugins/kuttl.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kuttl
+spec:
+  version: "v0.1.0"
+
+  shortDescription: Declaratively build, install, and run operators using KUTTL.
```

Comment:
If I'm not mistaken, this tool only works with `kudo` operators. That's something you should mention here as well. Otherwise, users will not find this with `krew search kudo` with the current version of krew.

Reply:
`kuttl` was created out of kudo to run any k8s workloads and any operators.  It was changed to NOT be kudo specific.  In a week or 2 when we have this "ready" we would like to put this in front of the sig testing as well.